test/integration/guest_user_test.rb

```diff
@@ -65,62 +69,71 @@ def setup
   end
 
   test "unregistered admin cannot see category" do
+    skip
     ApplicationController.any_instance.stubs(:current_user).returns(user_user)
     visit items_path
     refute page.has_content?("Create Category")
   end
 
-  # test "an unauthorised user can view a single items page" do
-  # item = Item.create(title: 'coffee', description: 'black nectar of the
-  # gods', price: 1200)
-  #
-  #   visit root_url
-  #   click_link_or_button('Menu')
-  #   click_link_or_button('coffee')
-  #
-  #   within('#title') do
-  #     assert page.has_content?('coffee')
-  #   end
-  #
-  #   within('#description') do
-  #     assert page.has_content?('black nectar of the gods')
-  #   end
-  #
-  #   within('#price') do
-  #     assert page.has_content?('$12.00')
-  #   end
-  # end
-
-  test "an unauthorised user can view a tenants page which only shows their
-  products" do
-    item = Item.create(title: "coffee",
-                       description: "black nectar of the gods", price: 1200)
-    item.categories(name: "agriculture")
-    create(:user)
-    create(:tenant)
-    # tenant.items < item
+  test "an unauthorised user can view a single items page" do
+    skip
+  item = Item.create(title: 'coffee', description: 'black nectar of the
```

Comment:
Useless assignment to variable - `item`.<br>Inconsistent indentation detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.